test/integration/tests/crud-entity/common-crud.test.js

```diff
@@ -149,22 +149,26 @@ function testFunc() {
         describe('findById()', () => {
           it('should throw a NotFoundError if the id argument does not exist', (done) => {
             const entityId = utilities.randomString();
-            storeToTest.findById(entityId).toPromise()
-              .then(() => done(new Error('Should not be called')))
-              .catch((error) => {
-                expect(error.name).to.contain(notFoundErrorName);
-                done();
-              })
-              .catch(done);
+            const nextHandlerSpy = sinon.spy();
```

Comment:
It seems that the tests timeouts if the assertion fails

Reply:
The test ''should throw a NotFoundError if the id argument does not exist''

Reply:
Wrapped in a try/catch to avoid timing out.